ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/WelcomeFrame.kt

```diff
@@ -40,6 +40,7 @@ class WelcomeFrame(remoteRobot: RemoteRobot, remoteComponent: RemoteComponent) :
             actionLink("Open").click()
         }
         fillSingleTextField(path.toAbsolutePath().toString())
```

Comment:
Can we hide this in a fixture? I presume we would have this logic followed by next line anytime we need to browse for files (i.e. S3 upload/download)

Reply:
Yep, replaced